containers/ws/release.sh

```diff
@@ -1,31 +1,36 @@
-#!/bin/sh
+#!/bin/bash
 set -eux
 
-if [ -z "${RUNC:-}" ]; then
-    RUNC=$(command -v podman || command -v docker) || {
-        echo "ERROR: podman or docker required" >&2
-        exit 1
-    }
+command -v podman || {
+    echo "ERROR: podman required" >&2
```

Comment:
So docker can't build manifests? Not that anyone in our team cares, but is there a reason to drop docker in this script?

Reply:
I just noticed, in the end, this script got changed less than expected and the podman-only stuff is only executed in the `manifest` job in the workflow, so I can re-add the `$RUNC` variable.

(Yes, Docker can build manifests as well, but AFAIK the command syntax is different)